components/brave_wallet_ui/components/desktop/views/market/index.tsx

```diff
@@ -0,0 +1,163 @@
+import * as React from 'react'
+
+import { SearchBar } from '../../../shared'
+import { AssetsFilterDropdown } from '../..'
+import { AssetFilterOptions } from '../../../../options/market-data-filter-options'
+
+import {
+  LoadIcon,
+  LoadIconWrapper,
+  StyledWrapper,
+  TopRow
+} from './style'
+import { useMarketDataManagement } from '../../../../common/hooks/market-data-management'
+import { marketDataTableHeaders } from '../../../../options/market-data-headers'
+import { AssetFilterOption, BraveWallet, MarketDataTableColumnTypes, SortOrder } from '../../../../constants/types'
+import { MarketDataTable } from '../../../market-datatable'
+import { debounce } from '$web-common/debounce'
```

Comment:
where is this dependency resolving to?

Reply:
@diracdeltas It is an alias for `components/common` as defined [here](https://github.com/brave/brave-core/blob/f42669ecc4199486adcb2637b4d97c8c0169c533/tsconfig.json#L60)